powerdns/models/requests.py

```diff
@@ -46,6 +47,7 @@ class Meta:
         null=True,
         blank=True
     )
+    last_change_json = models.TextField(null=True, blank=True)
```

Comment:
consider jsonfield for this (http://django-extensions.readthedocs.io/en/latest/field_extensions.html?highlight=json) - you're using django-extensions right now either way.